ddprof-lib/src/main/cpp/livenessTracker.cpp

```diff
@@ -44,44 +44,47 @@ void LivenessTracker::cleanup_table(bool forced) {
     JNIEnv* env = VM::jni();
     jvmtiEnv* jvmti = VM::jvmti();
 
-    u64 start = OS::nanotime(), end;
+    int epoch_diff = (int) (target_gc_epoch - current);
 
     _table_lock.lock();
-    int epoch_diff = (int) (target_gc_epoch - current);
```

Comment:
is this change necessary?

Reply:
The whole block change was triggered by division by zero in the debug statement. While that is unlikely, getting the table of size 0 is still possible and moving the check for the non-zero check before doing anything more expensive seemed like an ok idea.